client/src/components/History/ContentOperations.vue

```diff
@@ -143,6 +94,38 @@
                         <span v-localize>Build Collection from Rules</span>
                     </b-dropdown-item>
                 </b-dropdown>
+
+                <b-dropdown
+                    class="history-contents-list-action-menu-btn"
+                    size="sm"
+                    text="History"
+                    :disabled="!totalMatches"
+                    data-description="history action menu"
+                >
+                    <b-dropdown-text id="history-op-all-content">
+                        <span v-localize>With entire history...</span>
+                    </b-dropdown-text>
+
+                    <b-dropdown-item
+                        aria-describedby="history-op-new-content"
```

Comment:
Since `#history-op-new-content` is gone, please remove it.
```suggestion
```

Reply:
there are about 6 more places where `aria-describedby="history-op-new-content"` is still present

Reply:
Good catch; there are more of these that need to be adjusted to say 'selected' content, so I'll add that.